src/routers/users.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+
+let users = [];
+
+router.get('/', (req, res) => {
+  res.send(users);
+});
+
+router.get('/:userId', (req, res) => {
+  const { userId } = req.params;
+  const foundUser = users.find(user => user.id === +userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
```

Comment:
`return 404 with any message if expected entity doesn't exist.`

You've always sended a 404 error without the message